content/local/writer.go

```diff
@@ -58,6 +60,8 @@ func (w *writer) Status() (content.Status, error) {
 //
 // Cannot be called concurrently with `Write`.
```

Comment:
if we are managing concurrency with a mutex across these functions, then this comment should be changed/removed